app/assets/javascripts/modal/modal-workflow.js

```diff
@@ -11,9 +11,10 @@ ModalWorkflow.prototype.initComponents = function () {
 
 ModalWorkflow.prototype.overrideActions = function (actions) {
   var formItems = this.$modal.querySelectorAll('form[data-modal-action]')
-  var linkItems = this.$modal.querySelectorAll('a[data-modal-action]')
+  var linkItems = Array.prototype.slice.call(this.$modal.querySelectorAll('a[data-modal-action]'))
```

Comment:
I think we can do both at the same time, with `this.$modal.querySelectorAll('a[data-modal...], button[data-modal...]')`.

https://stackoverflow.com/questions/37552933/concat-two-nodelists